src/App.js

```diff
@@ -1,29 +1,19 @@
-import logo from './logo.svg';
+import React, { useState } from 'react';
+import Calculator from './components/Calculator';
+import calculate from './logic/calculate';
```

Comment:
[REQUIRED] You are expected to import `calculate` into your calculator component. If you notice it your way complicates things and involves a lot of passing props down. When the clicks could have simply been handled in the component. Kindly fix this.